examples/testers/es-github.yaml

```
@@ -0,0 +1,18 @@
+apiVersion: argoproj.io/v1alpha1
+kind: EventSource
+metadata:
+  name: argo-tester-github
+  namespace: devops
+  annotations:
+    v1alpha1.argoslower.kanopy-platform/known-source: "github"
+spec:
+  eventBusName: demo-days
```

nit. not sure about my suggestion either. Here and elsewhere for `demo-days`
```suggestion
  eventBusName: argoslower
```

---

Refactored but you will have to manually delete the existing resources and old event bus and recreate the event bus and event sources.

---

Easy, I had to rebuild me entire env.

---

oh woops sorry